### ISSUE TITLE
Some comment tags are adversely affected by format comments

### ISSUE THREAD

**Issue by uversy** [codecadwallader/codemaid#141]
The first one I've seen that is no good is when using the `<see>` tag, which is often used inline in comment text, splitting tags to multiple lines affects these tags.  They should be ignored and allowed to remain inline if there is other text on the same line as the `<see>` tag.

The other issue I'm seeing is in regards to the `<code>` tag.  Here is how it looks before cleaning up comments:
```
        /// <summary>
        ///     Get the total memory installed on the Server
        ///     (in GB)
        /// </summary>
        /// <returns>
        ///     Total memory as <see cref="System.UInt32" />
        /// </returns>
        /// <example>
        ///     Method usage: 
        ///     <code>
        ///         using libServerInfo.Collectors;
        ///         using System.Windows;
        ///         ...
        ///         OS myOS = new OS();
        ///         
        ///         MessageBox.Show("Total RAM:  " + totalRam);
        ///         
        ///         myOS = null;
        ///     </code>
        /// </example>
```
Here is how it looks after cleaning up comments:
```
        /// <summary>
        ///     Get the total memory installed on the Server
        ///     (in GB)
        /// </summary>
        /// <returns>
        ///     Total memory as <see cref="System.UInt32" />
        /// </returns>
        /// <example>
        ///     Method usage: 
        ///     <code>
        ///                 using libServerInfo.Collectors;
        ///                 using System.Windows;
        ///                 ...
        ///                 OS myOS = new OS();
        ///         
        ///                 MessageBox.Show("Total RAM:  " + totalRam);
        ///         
        ///                 myOS = null;
        ///         </code>
        /// </example>
```

Notice that everything below the `<code>` tag up to and including the `</code>` tag are shifted right?  They will tab right once for each cleanup.  Now, since I do comment cleanup on a file save, I'm now have code samples that are shifted so far to the right they aren't even on screen anymore.  This makes me sad.

**Comment by codecadwallader** [comment]
I am not able to reproduce what you are seeing.  Some questions:

1) Which version of Visual Studio are you using?
2) Which version of CodeMaid are you using?
3) What formatting configuration do have you specified in CodeMaid->Options->Formatting?
4) Have you changed Visual Studio to use tabs instead of spaces?  Are there any other configuration changes that jump to mind that may be relevant?

**Comment by codecadwallader** [comment]
Closing due to inactivity.  Please comment to reopen.

**Comment by Myrddraal** [comment]
I have a similar problem, again using <code> tags, but this time self-closing, as follows:

Before format comment:
        /// <example>
        ///     Example usage :
        ///     <code source="..\MyExamples\Examples.cs" region="Example1" language="cs" />
        ///     Example usage with a location parameter and a location function:
        ///     <code source="..\MyExamples\Examples.cs" region="Example2" language="cs" />
        /// </example>

After:
        /// <example>
        ///     Example usage :
        ///     <code source="..\MyExamples\Examples.cs" region="Example1" language="cs" />
        ///         Example usage with a location parameter and a location function:
        ///     <code source="..\MyExamples\Examples.cs" region="Example2" language="cs" />
        ///         </example>

I tried, but can't attach my codemaid settings. I'm using version 0.7.6.

Relevant settings:
Extra indent for values inside XML comments: 4
Add space inside self-closing tags.
Keep XML tags together.
Always split <summary> tags onto multiple lines.

**Comment by codecadwallader** [comment]
@Myrddraal Thank you for the additional details, I am now able to reproduce both issues.  It is specific to the setting for adding extra indentation inside XML comments, which is why I did not originally reproduce it.

**Comment by willemduncan** [comment]
Found the problem and fixed in b555ee4a079cec8088a977f4fdc26e8c606d310d, included unit tests to assure it stays that way.

**Comment by codecadwallader** [comment]
That looks great @willemduncan , thank you for addressing it so quickly!  One minor comment on using statement order but if you want to issue a pull request when you have a chance I'll get it merged in for the next release. :)

**Comment by Myrddraal** [comment]
That's great news, thanks. Because we use Codemaid and also enforce Stylecop, we currently have a running battle going between the two, with Codemaid regularly breaking the indentation, and Stylecop regularly complaining.

Thanks for fixing so fast! 
(and now to be impatient: do you have a rough idea when the next release might be?)

**Comment by codecadwallader** [comment]
Since this is a side project for everybody, we don't have any committed timelines.. but I'm roughly hoping to have the next major release out in the next month or so.  We will post betas along the way to that as the feature set stabilizes, so keep an eye on the downloads or join the beta testers mailing group (https://trello.com/c/5RFsLwow) for a preview.

**Comment by willemduncan** [comment]
I was wondering what happened with the `using ...` reordering, I did not do that intentionally, it resulted from habitually running the CodeMaid cleanup after working on any file. So we now sort all statements by alphabet rather than put `System` directives first? I will change my version to keep the PR to a minimum.

This once again makes a great argument for solution specific settings, which would have prevented this.

**Comment by codecadwallader** [comment]
Yep, MS changed the standard in VS2012.  For some more info see #75

**Comment by willemduncan** [comment]
I was aware of the setting but I always checked the `System` first option. I did not know the 'official' opinion on that changed. Updated the using statement orders in fb64ac5d4648e5ec0d84248968beafb8f0c73daf.

**Comment by codecadwallader** [comment]
Cool, thanks. :)

**Comment by Myrddraal** [comment]
Thanks, I'll be sure to keep an eye out for the beta.

**Comment by codecadwallader** [comment]
This is present in the latest release, https://github.com/codecadwallader/codemaid/releases/tag/v0.8.8-alpha.  If you have the chance to test it and comment that would be greatly appreciated!

**Comment by Myrddraal** [comment]
Thanks, I've downloaded and tested. The issue as described is fixed, but there does appear to be one issue remaining. Taking the same code sample as above, the result is now as follows:

Before format comment:
```
        /// <example>
        ///     Example usage :
        ///     <code source="..\MyExamples\Examples.cs" region="Example1" language="cs" />
        ///     Example usage with a location parameter and a location function:
        ///     <code source="..\MyExamples\Examples.cs" region="Example2" language="cs" />
        /// </example>
```

After:
```
        /// <example>
        ///     Example usage :
        ///     <code source="..\MyExamples\Examples.cs" region="Example1" language="cs" />
        /// Example usage with a location parameter and a location function:
        ///     <code source="..\MyExamples\Examples.cs" region="Example2" language="cs" />
        /// </example>
```

As you can see, after the first <code> tag, the next line of text is not indented. 

A further example shows that it is only the line after the tag which is affected:

...
        /// <example>
        ///     Example usage :
        ///     <code source="..\MyExamples\Examples.cs" region="Example1" language="cs" />
        /// Example usage with a location parameter and a location function:
        ///     <code source="..\MyExamples\Examples.cs" region="Example2" language="cs" />
        /// Test text Text textTest text Text textTest text Text textTest text Text textTest text Text textTest text Text
        ///     textTest text Text textTest text Text textTest text Text textTest text Text textTest text Text textTest text
        ///     Text textTest text Text textTest text Text text
        /// </example>
...

**Comment by codecadwallader** [comment]
Thanks for retesting, reopening issue.

**Comment by willemduncan** [comment]
I see, I misunderstood this because I never saw self-closing code tags before. Will create a change soon to add support.

**Comment by willemduncan** [comment]
This should be resolved with the latest commit ee61cd286163b3bcd4309cf02017ab123d670629, just a few smal things to wrap up and then I'll send this off in a PR to the main version.
